accesslog.go

```diff
@@ -0,0 +1,79 @@
+package transport
+
+import (
+	"net"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/asecurityteam/logevent"
+)
+
+type accessLog struct {
+	Time                   string   `logevent:"@timestamp"`
+	UGCDirty               []string `logevent:"ugc_dirty"`
```

Comment:
Does this struct represent everything we have access to in the request? Why are some fields defined here (like UGCDirty and HTTPContentType), but not set in RoundTrip?

Reply:
Yeah, I think the idea there was something we were going to use but never did. I removed it.